### DISCUSSION TITLE
Our August 2024 monthly call: community choice!

### DISCUSSION THREAD

**Discussion by finekatie** [uswds/uswds#5923] [General]
> **Community update — September 17, 2026:** This is a historical monthly-call post. For current questions and participation, use [USWDS GitHub Discussions](https://github.com/uswds/uswds/discussions). Recordings remain available in the [monthly call archive](https://designsystem.digital.gov/about/monthly-calls/). References below to Slack, staff roles, or future calls reflect the community at the time.

This August, we want to know what you want to hear! What do you need to know about the design system? Which monthly call topics would help your team build more human-centered, accessible projects? _What should we be talking about?_

This'll be our first community choice monthly call, and we need to hear from _you_ about what you need to hear from _us_.

We have a few ideas, but these are just a starting point. Perhaps the most useful topics are the ones we hadn't considered — maybe the one that you contribute!

- Browser support  
- Product roadmap
- Tools for designers
- Oops, all Q&A!
- If we had a magic wand…
- WCAG 2.2 and beyond

**Comment by smcharg-ohss** [comment]
I'd be interested in an update on the process for getting new components added to USWDS.

**Reply by mgifford** [reply]
I'd be very interested in this as well. I added a submission here https://github.com/uswds/uswds-proposals/pull/8

Not sure if this is the right way to do it, or what will happen now that I've submitted it. 

Also, what do we do with things, like Dark Mode that aren't really components, but rather approaches?

**Comment by sly-tanenbaum** [comment]
Figma support! That would be lovely.

**Comment by hcorson-dosch-usgs** [comment]
I'd be interested in an overview/update on using USWDS components with Vue3

**Comment by dplundberg** [comment]
Our Agency is migrating internal web apps and associated public-facing web sites from a custom .NET solution to Dynamics 365/Power platform solutions. What are the recommendations/guidance for agencies that will be developing public-facing sites leveraging Power Pages over the course next year and beyond?

**Comment by brunerae** [comment]
> **Community update — September 17, 2026:** The Slack guidance below is historical. For current questions and participation, use [USWDS GitHub Discussions](https://github.com/uswds/uswds/discussions). Past calls remain available in the [monthly call archive](https://designsystem.digital.gov/about/monthly-calls/).

Thanks for all of your questions and inputs! We'll be addressing these contributions in our call on August 15. Be on the lookout for information about how to register for the event in our #uswds-public Slack channel and in this month's newsletter. Hope you'll join us!
